compliance/store/src/test/java/org/eclipse/rdf4j/sail/memory/FastMemInferencingTest.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Also wrong copyright header. This is consistently the case for all new files you created. See the eclipse templates in eclipse-settings for the correct header.